### PR TITLE
Add Use Cases and Glossary Terms

### PR DESCRIPTION
**Pull request by syy94**
Added Use Cases(Add User, Edit User, Export Address Book). Also added 2 new Glossary terms and updated affected existing terms.

### CODE REVIEW

**Comment by kengying** [comment]
for add person, there should be a step three for, AddressBook shows confirm message.

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -795,6 +870,19 @@ Private contact detail
...
 ....
 
+[[tag]]
+Tag
+
+....
+A 'tag' is a temporary label that can be used as a casual, mental-note for users to quickly remember who the contact is. For instance, a tag can include detailed relationships ('BFF') or short description about that person ('cool professor').
+....
+
+[[Group]]
```
Comment (sofarsophie) [inline docs/DeveloperGuide.adoc]: Can the group tag be in lowercase? i.e., [[group]]

**Comment by sofarsophie** [comment]
- [[Group]] label should be written in lowercase ([[group]])
- link:tag[tag] => missing a # in a few lines (link:#tag[tag])

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (syy94) [reply docs/DeveloperGuide.adoc]: its a typo, will be fixed in the next commit

**Comment by syy94** [comment]
@kengying The third step is added

**Pull request merged**